docs/conf/eu/2017/news/recap-retrospective.rst

```diff
@@ -0,0 +1,90 @@
+:template: 2017/eu-content.html
+
+.. post:: Oct 16, 2017
+   :tags: 2017, prague, thank you, feedback
+
+Write the Docs Prague 2017 - Recap and Retrospective
+====================================================
+
+Hi everyone,
+
+Thanks for attending the 2017 edition of Write the Docs Prague! We're so grateful to our speakers, sponsors, volunteers, and attendees for making the conference possible.
+
+We hope you had a wonderful time, and that you are as excited as we are about the beautiful community that gathers under our roof. Below we've included all of the resources you might need for staying in touch and continue to build and grow the community.
+
+We also have a Code of Conduct transparency report and information about feedback to help make the conference even better next year.
```

Comment:
'We've also included' to make it clearer that it's attached.

Reply:
I'm not very keen on the link from "CoC report and feedback" to "make the conf better". First part is for everyone, second part is mostly on us. Discuss? :-p

Reply:
I think they're both relevant for everyone, silly emoji faces aside. The CoC report is there to help people know that shit happens and they can speak up next year, and the feedback is feedback...

Reply:
Yeah, I know. But I still find the direct link too much.

Reply:
In that case, the silly emoji face is confusing because it makes it seem like your comment can be taken more lightly :P 

Do you have an alternative phrasing?

Reply:
We also have a Code of Conduct transparency report, and a summary of your feedback which we'll use to improve the conference next year. 

meh, not too keen on that either.

Reply:
Nevermind, I'm fixing it now.